content/en/metrics/_index.md

```diff
@@ -138,45 +138,45 @@ After selecting a metric, you can filter your query based on tag(s). For instanc
 
 #### Configure time aggregation
 
-Next, choose the granularity of your data using time rollup. In this example, you've defined that there is one data point for every hour (3600 seconds). You can choose how you want to aggregate the data in each time bucket. By default, _avg_ is applied, but other available options are _sum_, _min_, _max_, and _count_. You can also customize how your metrics data is aggregated and bucketed with functions or in-application modifiers. For example, if you wanted to apply max and customize how your metrics data is rolled up and bucketed in time with calendar aligned queries, you would use `.rollup(max, 60)`. For more information, see the [Functions][24], [Rollup][23], and [In-application modifiers][25] documentation.
+Next, choose the granularity of your data using time rollup. In this example, you've defined that there is one datapoint for every hour (3600 seconds). You can choose how you want to aggregate the data in each time bucket. By default, _avg_ is applied, but other available options are _sum_, _min_, _max_, and _count_. You can also customize how your metrics data is aggregated and bucketed with functions or in-application modifiers. For example, if you want to apply _max_ and customize how your metrics data is rolled up and bucketed in time with calendar aligned queries, you can use `.rollup(max, 60)`. For more information, see [Functions][24], [Rollup][23], and [In-application modifiers][25].
```

Comment:
"For example, if you want to apply _max_ and customize how your metrics data is rolled up and bucketed in time with calendar aligned queries, you can use `.rollup(max, 60)`."

What does this mean in more basic English?